src/enums/data-location.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable no-shadow */
+/* eslint-disable no-unused-vars */
+export enum DataLocation {
+    BLOB_STORAGE = 'BlobStorage',
+    DATA_LAKE = 'Datalake'
+}
```

Comment:
Could you add
```suggestion
    UNKNOWN = 'Unknown'
```
Turns out there is a need for an unknown location especially when testing things

Reply:
done 👍